Cargo.toml

```diff
@@ -396,6 +396,17 @@ description = "Renders a 2d mesh"
 category = "2D Rendering"
 wasm = true
 
+[[example]]
+name = "mesh2d_circular"
+path = "examples/2d/mesh2d_circular.rs"
+doc-scrape-examples = true
+
+[package.metadata.example.mesh2d_circular]
+name = "Circular 2D Meshes"
+description = "Demonstrates UV-mapping of circular primitives"
```

Comment:
I'm wondering if there's a better name... "circular" might imply that this also includes circles and maybe ellipses, but it just has the arc-like shapes. `mesh2d_circular` and `mesh2d_arcs` would both be fine probably

Reply:
I like `_arcs` better.